packer/plugin/discover.go

```diff
@@ -9,6 +10,7 @@ import (
 	"sort"
 	"strings"
 
+	"github.com/hashicorp/packer/packer-plugin-sdk/packer"
```

Comment:
```suggestion
```
This is defined already bellow with `packersdk`

Reply:
Woops, right, I didn't look at this conflict in the eyes 🙂